.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence.
-*		    @GoogleCloudPlatform/anthos-dpe @bgrant0607 @haiyanmeng @janetkuo @karlkfi @Liujingfang1 @morgante @nan-yu @nicjjohns @selfmanagingresource
\ No newline at end of file
+*		    @GoogleCloudPlatform/anthos-dpe @haiyanmeng @janetkuo @karlkfi @nan-yu @selfmanagingresource
```

Comment:
I am not sure if some of these Googlers should be removed (or preserved) as `CODEOWNERS`. But since the contributions from the `CODEOWNERS` you're removing are from 2021, I approve these removals. They can of course ask us to add them back in (if necessary). :)

Reply:
Ah, just saw your comment about the email you sent to the `CODEOWNERS` you're removing. Thanks!

Reply:
The ones I removed are either a) no longer at Google or b) have personally told me to remove them
Still pending on any potential other removals, but they are done intentionally :)